.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - lts/*
+cache:
+  directories:
+    - "~/.npm"
+before_install:
+  - npm install -g npm@latest
+  - npm install -g greenkeeper-lockfile@1.14.0
```

Comment:
Are we officially using greenkeeper and is there anything additional we need to do to configure?